pages/pipelines/command_step.md.erb

```diff
@@ -201,7 +201,7 @@ _Optional attributes:_
   <tr id="timeout_in_minutes">
     <td><code>timeout_in_minutes</code></td>
     <td>
-      The maximum number of minutes a job created from this step is allowed to run. If the job exceeds this time limit, or if it finishes with a non-zero exit status, the job is automatically canceled and the build fails. Jobs that time out with an exit status of 0 are marked as "passed". Note that command steps on the Buildkite <a href="https://buildkite.com/pricing">Free plan</a> have a maximum job timeout of 240 minutes.<br>
+      The maximum number of minutes a job created from this step is allowed to run. If the job exceeds this time limit, or if it finishes with a non-zero exit status, the job is automatically canceled and the build fails. Jobs that time out with an exit status of 0 are marked as "passed".<br> To prevent jobs from consuming too many job minutes or running forever, you can also specify default and maximum timeouts from your organization's <a href="https://buildkite.com/organizations/~/pipeline-settings">Pipeline Settings page</a>, or on a pipeline's Builds settings page. Default timeouts apply in order of specificity — step's, pipeline's, and finally the organization's. Maximum timeout will be applied to any command step without a timeout or with a timeout greater than the maximum timeout.<br> Note that command steps on the Buildkite <a href="https://buildkite.com/pricing">Free plan</a> have a maximum job timeout of 240 minutes.<br>
```

Comment:
This makes it sound like default and maximum are different things, but they aren't.

Reply:
Regarding specificity, I'm not quite clear on the intention.

Reply:
But they are different things.
![Uploading specify-timeout.png…]()

Reply:
Ahah, yes, in *that* case we need to be more clear about this. :-p